test/Assert.t.sol

```diff
@@ -28,28 +28,26 @@ contract TestAsserts is Test, HelperAssert {
         t(false, reason); 
     }
 
-    function test_HelperAssert_eq_x_x_concrete() public {
+    function test_eq_x_x() public {
         uint256 x = 1;
-        eq(x, x, "eq does not revert with equal values");
+        eq(x, x, "example message");
     }
 
-    function testFuzz_HelperAssert_eq_x_x_fuzz(uint256 x) public {
-        eq(x, x, "eq does not revert with the fuzz values");
+    function testFuzz_eq_x_x(uint256 x) public {
+        eq(x, x, "example message");
     }
 
-    function test_HelperAssert_eq_x_y_concrete() public {
+    function test_eq_x_y() public {
         uint256 x = 2;
         uint256 y = 4;
 
-        string memory reason = "eq reverts with different values.";
-        string memory failReason = string(abi.encodePacked(
-                "Invalid: ",
+        string memory reason = "example message";
+        string memory failReason = string(abi.encodePacked(createAssertFailMessage(
```

Comment:
createAssertFailMessage should return a `string` instead of bytes, then you can remove `string(abi.encodePacked(` here

this also needs some refactoring in `HelperAssert.sol`